src/content/docs/en/user-guide/bridge.mdx

```diff
@@ -20,32 +20,25 @@ excerpt: "To start bridging assets from Sepolia, navigate to the portal bridge a
 {/* TODO: Update all instructions after being able to walk through the whole flow. */}
 
 
-Visit our [Bridge](https://sepolia.scroll.io/bridge) app to get started![^thanks-hop] The Bridge supports both **Deposit** and **Withdraw** operations, allowing users to trustlessly move assets from Sepolia Testnet to Scroll Sepolia Testnet.
+Visit our bridge app on [Sepolia](https://sepolia.scroll.io/bridge) or [Mainnet](https://scroll.io/bridge) to get started![^thanks-hop] The Bridge supports both **Deposit** and **Withdraw** operations, allowing users to trustlessly move assets from L1 to L2.
 [^thanks-hop]: Forked from [Hop Exchange](https://hop.exchange/)'s UI 🙌
 
-Deposits can take up to 15 minutes to be available on Scroll.
-
-Withdrawals, which require a second interaction on Sepolia after the withdrawal is finalized, can take much longer.
-
-:::caution[Waiting a long time for a bridge transaction?]
-The time estimates above are typical for normal network behavior and activity levels. Because we only process so many queued L1 transactions per an L2 block, bridged messages may take longer to be included on Scroll in times of exceptional network usage.
-:::
-
-## Deposit from Sepolia to Scroll Sepolia
+## Deposit from L1 to L2
```

Comment:
Given the audience for the Getting Started section (end users), I think we might just say "Deposit from Ethereum to Scroll" and assume test net users will figure it out.